### PR TITLE
Do not crash if nested plugin does not call next

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/fastify/fastify/issues/5410

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by gurgunday — APPROVED** [review]
lgtm

**Review by Uzlopak — APPROVED** [review]
Makes sense

File: test/plugin-timeout.test.js [hunk 1]
```diff
@@ -195,3 +195,45 @@ test('timeout without calling next in ready and rethrowing the error', (t) => {
...
+    app.use(two).after(next)
+  }
+
+  function two (app, opts, next) {
+    // do not call next on purpose
+  }
+  app.ready((err) => {
+    t.ok(err)
+    t.equal(err.fn, one)
+    t.equal(err.message, message('one')) // TODO: should be 'two'
```
Comment (gurgunday) [inline test/plugin-timeout.test.js:215]: So this means we should in the future be able to show that two failed right?

**Review by ivan-tymoshenko — APPROVED** [review]

**Comment by mcollina** [comment]
@gurgunday @ivan-tymoshenko PTAL. I fixed the error message too.

**Review by gurgunday — APPROVED** [review]

**Pull request merged**